### PR TITLE
FIO-9008 Added some of the missing accessibility tags to bootstrap5

### PR DESCRIPTION
**Pull request by antonSoftensity**
https://formio.atlassian.net/browse/FIO-9008

### CODE REVIEW

File: src/templates/bootstrap5/file/form.ejs [hunk 1]
```diff
@@ -119,7 +119,7 @@
         <span>
           <img ref="fileImage" src="" alt="{{file.originalName || file.name}}" style="width:{{ctx.component.imageSize}}px">
           {% if (!ctx.disabled) { %}
-            <i tabindex="0" class="{{ctx.iconClass('remove')}}" ref="removeLink"></i>
+            <i tabindex="0" class="{{ctx.iconClass('remove')}}" ref="removeLink" aria-label="Remove button. Press to remove {{file.originalName || file.name}} file"></i>
```
[the remark this answers is not in the document]
Comment (antonSoftensity) [reply src/templates/bootstrap5/file/form.ejs:122]: @brendanbond  as far as I know, `ctx.t()` only works with static strings. Will it work with a string that has variables like it has in that case?

[the remark this answers is not in the document]
Comment (brendanbond) [reply src/templates/bootstrap5/file/form.ejs:122]: Yup, it takes interpolation context as a second arg, e.g. see [here](https://github.com/formio/formio.js/blob/66d554e28589e1be222d12fdad530a4d4acff164/src/components/file/File.js#L779)

File: src/templates/bootstrap5/file/form.ejs [hunk 2]
```diff
@@ -119,7 +119,7 @@
         <span>
           <img ref="fileImage" src="" alt="{{file.originalName || file.name}}" style="width:{{ctx.component.imageSize}}px">
           {% if (!ctx.disabled) { %}
-            <i tabindex="0" class="{{ctx.iconClass('remove')}}" ref="removeLink"></i>
+            <i tabindex="0" class="{{ctx.iconClass('remove')}}" ref="removeLink" aria-label="{{ctx.t('Remove button. Press to remove ' + (file.originalName || file.name) + ' file')}}"></i>
```
Comment (brendanbond) [inline src/templates/bootstrap5/file/form.ejs:122]:
```suggestion
            <i tabindex="0" class="{{ctx.iconClass('remove')}}" ref="removeLink" aria-label="{{ctx.t('Remove button. Press to remove {{fileName}}', { fileName: file.originalName || file.name })}}"></i>
```

**Review by brendanbond — CHANGES_REQUESTED** [review]
see comment

File: src/templates/bootstrap5/file/form.ejs (hunk 2 above)
Comment (antonSoftensity) [reply src/templates/bootstrap5/file/form.ejs:122]: @brendanbond I'm not sure we can use nested params inside of {{ }}, e.g. `{{ ctx.t(' {{ fileName }} ',  { fileName: foo }) }}` is not supported by ejs templates

**Review by brendanbond — APPROVED** [review]

**Pull request merged**